votor/src/certificate_pool.rs

```diff
@@ -555,11 +556,13 @@ impl CertificatePool {
     /// the size of the certificate
     #[cfg(test)]
     pub fn slot_notarized_fallback(&self, slot: Slot) -> Option<usize> {
+        // TODO: remove this line once `CertificateMessage` is moved into the repo`
+        use crate::certificate_pool::vote_certificate::get_vote_count;
         self.completed_certificates
             .iter()
             .find_map(|(cert_id, cert)| {
                 matches!(cert_id, CertificateId::NotarizeFallback(s,_,_) if *s == slot)
-                    .then_some(cert.vote_count())
+                    .then_some(get_vote_count(cert))
```

Comment:
We don't need to return vote count here as I mentioned above, that test only checks if the certificate exists.